python/generate-map/main.py

```diff
@@ -0,0 +1,32 @@
+from PIL import Image
+import requests
+from io import BytesIO
+from base64 import b64encode
+def error(res,message):
+    return res.json({"success":False,"error":message})
+def main(req,res):
+     
+     try:
+        payload = req.payload
+        lng = payload["lng"]
+        lat = payload["lat"]
+        access_token = req.env["token"]
+     except Exception:
+        return error(res,"The payload must contain lng and lat")
+     if(lng == None or lat == None):
+        return error(res,"The payload must contain lng and lat")
+     elif((type(lng)!=float or type(lat)!=float) and (type(lng)!=int or type(lat)!=int)):
+        return error(res,"The longitude and latitude must be a number")
+     elif(lng >180  or lng < -180 ):
```

Comment:
```suggestion
     elif(lng >180  and lng < -180 ):
```
it should be `and` right?

Reply:
ah mb